Lesson1/main.js

```diff
@@ -0,0 +1,118 @@
+// Урок 1
+
+// Задание 1
+
+function loop(times = 0, callback = null) {
+
```

Comment:
```suggestion
if (typeof callback === 'function') {
```
в задании требовалась проверка на передачу функции:)

Reply:
Спасибо за комментарий. Исправил.